src/scripts/modules/media/body/body.coffee

```diff
@@ -152,6 +152,27 @@ define (require) ->
             # NOTE: `.toggleClass()` explicitly requires a `false` (not falsy) 2nd argument
             $el.toggleClass('ui-has-child-title', $title.length > 0)
 
+          # Create subtitles in each chapter with corresponding ids as their href
+          $totalSubtitle = $temp.find("[data-depth='1']")
+          $totalId = $totalSubtitle.children('[data-type="title"]')
+          idLength = $totalId.length
+          countIndex = 0
+          realHref = window.location.pathname
+          #Determine if this is a outer page view or not
+          $location = $(".table-of-contents [href$='#{realHref}'] > .title")
```

Comment:
Was the conclusion from #1545 that these elements were going to be added to the Table of Contents (Tree) Model and use the existing Tree UI rather than recreating the UI from within here?